src/core/connection.c

```diff
@@ -3059,6 +3059,17 @@ QuicConnProcessPeerTransportParameters(
             Connection->Stats.GreaseBitNegotiated = TRUE;
         }
 
+        if (Connection->Settings.ReliableResetEnabled &&
+            (Connection->PeerTransportParams.Flags & QUIC_TP_FLAG_RELIABLE_RESET_ENABLED) > 0) {
+            //
+            // Reliable Reset is enabled on both sides.
+            //
+            Connection->State.ReliableResetStreamNegotiated = TRUE;
+        } else {
+            Connection->State.ReliableResetStreamNegotiated = TRUE;
```

Comment:
Remove this line.

Reply:
Actually, I think you should remove the whole else; just like Grease bit above.

Reply:
Shouldn't we also want to update the app's internal settings to be false?

Reply:
No. That flag reflects what the app wants. The fact that the feature wasn't negotiated doesn't change the fact that the app wanted it.